docker/ray-deps/Dockerfile

```diff
@@ -14,8 +14,8 @@ RUN $HOME/anaconda3/bin/pip --no-cache-dir install $(basename $WHEEL_PATH)[all]
         "google-api-python-client==1.7.8" \
         "google-oauth" \
         "kubernetes" \
-        "azure-cli-core==2.4.0" \
-        "azure-mgmt-compute==12.0.0" \
+        "azure-cli-core==2.22.0" \
+        "azure-mgmt-compute==20.0.0" \
         "azure-mgmt-msi==1.0.0" \
-        "azure-mgmt-network==10.1.0"; fi) \
+        "azure-mgmt-network==18.0.0"; fi) \
```

Comment:
need to roll back mgmt-compute and mgmt-network for all files to following verisions

```
    "azure-cli-core==2.22.0"
    "azure-mgmt-compute==14.0.0" 
    "azure-mgmt-msi==1.0.0"
    "azure-mgmt-network==10.2.0"
```